docker/package/test-ubuntu-binaries.py

```diff
@@ -0,0 +1,78 @@
+# SPDX-FileCopyrightText: 2021 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
+    "tezos-admin-client",
+    "tezos-node",
+    "tezos-signer",
+    "tezos-codec",
+    "tezos-baker-PtMumbai",
+    "tezos-accuser-PtMumbai",
+    "tezos-smart-rollup-client-PtMumbai",
+    "tezos-smart-rollup-node-PtMumbai",
+    "tezos-baker-PtNairob",
+    "tezos-accuser-PtNairob",
+    "tezos-smart-rollup-client-PtNairob",
+    "tezos-smart-rollup-node-PtNairob",
+]
+
+
+def setup():
+    commands = [
+        "apt update -y",
+        "apt install -y software-properties-common",
+        "apt update -y",
+        "add-apt-repository -y ppa:serokell/tezos",
+        "apt-get update -y",
+    ]
+    for command in commands:
+        result = subprocess.run(command, shell=True)
+        assert result.returncode == 0, f"{result.returncode}"
+
+
+def test(cleanup: bool):
+    sys.stdout.write("Setting up env\n")
+    setup()
+    sys.stdout.write("Setup is done\n")
+
+    processed_binaries = []
+    try:
+        for binary in binaries:
+            sys.stdout.write(
+                f"---------------------|Testing {binary}|---------------------\n"
+            )
+            sys.stdout.write(f"Installing {binary}\n")
+
+            install_command = f"sudo apt install -y {binary}"
+            installation_result = subprocess.run(install_command, shell=True)
+            assert installation_result.returncode == 0
+            sys.stdout.write(
+                f"{binary} binary successfully installed. Checking that version works fine\n"
+            )
+
+            processed_binaries.append(binary)
+
+            check_binary_command = f"sudo {binary} --version"
+            check_binary_result = subprocess.run(check_binary_command, shell=True)
+            assert check_binary_result.returncode == 0
+            sys.stdout.write(f"{binary} binary works fine\n")
+    except Exception as e:
+        print("Exception happened when trying to execute tests.\n")
+        print(f"{str(e)}")
```

Comment:
Script does not signal any error if something happens. As you can see in your GA logs, exception happened, but job was marked as successful.